docs/index.asciidoc

```diff
@@ -300,8 +300,7 @@ Refer to <<plugins-{type}s-{plugin}-ecs_metadata,ECS mapping>> for detailed info
                       explicitly specified, _excluding_ `codec_metadata` from `enrich` will
                       disable `ecs_compatibility` for this plugin.
 | source_metadata   | Information about the _source_ of the event, such as the IP address
-                      of the inbound connection this input received the event from and the
-                      name of the Logstash host that processed the event
+                      of the inbound connection this input received the event from
```

Comment:
Good catch, thanks for this.
Can we also bump up version (in `VERSION` file) and add changelog (in `CHANGELOG.md`)?
[Example change](https://github.com/logstash-plugins/logstash-input-beats/pull/474/files)